src/stylus/components/_rating.styl

```diff
@@ -0,0 +1,10 @@
+.v-rating
+  .v-icon
+    padding: 0.5rem
+    border-radius: 50%
+    user-select: none
+
+
+  &--readonly
+    .v-icon
+      cursor: default !important
```

Comment:
maybe `.v-rating { pointer-events: none }` to avoid `!important`?

Reply:
done